spec/spec_helper.rb

```diff
@@ -24,6 +24,12 @@
 require 'rsolr'
 require 'blacklight'
 
+require 'rspec-benchmark'
+
+RSpec.configure do |config|
+  config.include RSpec::Benchmark::Matchers
```

Comment:
Would it be faster to load individual tests if we load this in the individual test files, rather than for the whole suite? Example from [gem documentation](https://github.com/piotrmurach/rspec-benchmark):

```ruby
RSpec.describe "Performance testing" do
  include RSpec::Benchmark::Matchers
end
```